test/integration/test_hookcmds.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2025 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from typing import Callable
+
+import jubilant
+import pytest
+
+
+@pytest.fixture
+def juju_post_deploy(build_relation_charm: Callable[[], str], juju: jubilant.Juju):
```

Comment:
The name of this fixture (and how we're using it below) makes usage awkward, with `juju_post_deploy.run(...)` instead of `juju.run(...)`.

Can we instead call this fixture `pack_and_deploy` or something, and then the tests would depend on both `juju` and `pack_and_deploy`? (And there's some pytest way to ask for a fixture without making it an arg.) Alternatively make it an autouse fixture if appropriate.